licenses/LICENSE-paranamer.txt

```diff
@@ -0,0 +1,28 @@
+[ ParaNamer used to be 'Pubic Domain', but since it includes a small piece of ASM it is now the same license as that: BSD ]
```

Comment:
This LICENSE file has a .txt suffix and others do not. Consider removing the .txt suffix to keep things uniform.